README.md

```diff
@@ -1,2 +1,18 @@
-# HelloMicroverse-project
- Seting up a "Hello Microverse" project
+# Seting up a "Hello Microverse" project
```

Comment:
- [ ] [REQUIRED] Please also use the [Microverse README template]https://github.com/microverseinc/readme-template) but remember to customize it to your own project